resources/eventing/charts/event-publisher-proxy/templates/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "event-publisher-proxy.fullname" . }}
+  labels:
+  {{- include "event-publisher-proxy.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+  {{ include "event-publisher-proxy.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+    {{ include "event-publisher-proxy.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ include "event-publisher-proxy.serviceAccountName" . }}
+      containers:
+        - env:
+            - name: CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "event-publisher-proxy.secretName" . }}
+                  key: client-id
+            - name: CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "event-publisher-proxy.secretName" . }}
+                  key: client-secret
+            - name: TOKEN_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "event-publisher-proxy.secretName" . }}
+                  key: token-endpoint
+            - name: EMS_PUBLISH_URL
```

Comment:
Does it make sense to remove the EMS prefix?

Reply:
Since client-id, client-secret and token-endpoint also don't have it I think yes it makes sense. But that needs a code change as well.

Reply:
As agreed, will be tackled in another PR or if needed a ticket will be created :)